src/Kdyby/Redis/ConnectionPool.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+/**
+ * This file is part of the Kdyby (http://www.kdyby.org)
+ *
+ * Copyright (c) 2008 Filip Procházka (filip@prochazka.su)
+ *
+ * For the full copyright and license information, please view the file license.txt that was distributed with this source code.
+ */
+
+namespace Kdyby\Redis;
+
+use Kdyby;
+use Nette;
+
+
+
+/**
+ * @author Filip Procházka <filip@prochazka.su>
+ */
+class ConnectionPool extends Nette\Object
```

Comment:
I don't like the name very much, because it's not accurate. I don't have an idea for a better one through...